drivers/wireless/ieee80211/bcm43xxx/cyw_chip_43439.c

```diff
@@ -27,25 +27,34 @@
 
 #include "bcmf_interface.h"
 
+#include "../../../cyw43439.nvram.image"
```

Comment:
can we load image from storage?

Reply:
How other people replace nvram.image? At least, all firmware related stuff should put inside the board directory as c array or normal files.